assets/js/modules/reader-revenue-manager/datastore/service.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * `modules/reader-revenue-manager` data store.
+ *
+ * Site Kit by Google, Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { addQueryArgs } from '@wordpress/url';
+
+/**
+ * Internal dependencies
+ */
+import { createRegistrySelector } from 'googlesitekit-data';
+import { CORE_USER } from '../../../googlesitekit/datastore/user/constants';
+
+const selectors = {
+	/**
+	 * Returns a link to the Reader Revenue Manager platform.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param {string} publicationID Publication ID.
+	 * @return {string} Link to Reader Revenue Management platform.
+	 */
+	getServiceURL: createRegistrySelector(
```

Comment:
A couple points about the selector:

1. In the DocBlock, it says that it accepts a `publicationID` parameter, but in reality, it doesn't appear to do so. This should be accepted as an optional parameter and added to the URL if defined.
2. The arguments object (containing `path` and `query`) are not included in the DocBlock. Please see `assets/js/modules/analytics-4/datastore/service.js` for example.

Reply:
The docblock has been updated. I think as you added this comment on line 

```js
getServiceURL: createRegistrySelector(
```

which is still unchanged thus GH is showing the this has not been addressed. 

Can you please look at the docblock and let me know if it is fine?

Thanks